src/components/Form/Form.js

```diff
@@ -72,6 +74,8 @@ export default class Form extends PureComponent {
   static Switch = Switch;
   static Errors = Errors;
   static Success = Success;
+  static Section = Section;
+  static Description = Description;
```

Comment:
Created these two components to have some kind of Styled Components so that we can reuse to style our forms.